src/components/icon/icons/calculator.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Calculator({
+  stroke,
+  strokeWidth,
+  style: styleProp,
+  width,
+  height,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+
+  return (
+    <svg width={width} height={width} style={style} {...rest} viewBox={`0 0 36 36`}>
+      <g stroke="none" strokeWidth={strokeWidth} fill="none" fillRule="evenodd">
+        <g fillRule="nonzero">
+          <path d="M17.1,14.9c0,1.1-0.9,2-2,2H2c-1.1,0-2-0.9-2-2V2c0-1.1,0.9-2,2-2h13.1c1.1,0,2,0.9,2,2V14.9z" fill="#FFFFFF" />
+          <path d="M36,14.9c0,1.1-0.9,2-2,2H20.9c-1.1,0-2-0.9-2-2V2c0-1.1,0.9-2,2-2H34c1.1,0,2,0.9,2,2 L36,14.9L36,14.9z" fill="#FFFFFF" />
+          <path d="M17.1,33.6c0,1.1-0.9,2-2,2H2c-1.1,0-2-0.9-2-2V20.6c0-1.1,0.9-2,2-2h13.1c1.1,0,2,0.9,2,2 V33.6z" fill="#FFFFFF" />
+          <path
+            d="M36,33.6c0,1.1-0.9,2-2,2H20.9c-1.1,0-2-0.9-2-2V20.6c0-1.1,0.9-2,2-2H34c1.1,0,2,0.9,2,2 L36,33.6L36,33.6z"
+            fill="#FFFFFF"
```

Comment:
`fill` should not be hardcoded to `#FFFFFF` (none of them, there are multiple occurances), rather use `fill={fill}` instead.